doc/rst/source/changes.rst

```diff
@@ -4,6 +4,72 @@
+#. New :doc:`animation 13 <animations/anim13>` of seismic waveforms.
+#. Support for **+a**\ *angle* for y-axis as well as x-axis with the :ref:`-B axes settings <gmt:Axes settings>`.
+#. General improvements to the automatic determination of frame attributes.
+#. New **+d**\ *divisor* modifier to the :ref:`-i option <gmt:The **-i** option>` to simplify scaling of input values.
+#. Allow parsing of **-Jz**\ *1:zzzzzz* for vertical scale.
+#. New GMT configuration parameters :term:`MAP_FRAME_PERCENT`, :term:`COLOR_SET`, and :term:`COLOR_CPT`.
+#. Add support for reading variable in NetCDF-4 groups.
+#. Allow specifying the reciprocal increment for generating 1d arrays.
+#. Allow Latex expressions in single-line titles and Cartesian axes labels, add support for multi-line plot titles, and
+   add support for subtitles.
+#. Many general documentation improvements.
+#. API improvements to support the GMT/MEX, PyGMT, and GMT.jl environments.
+
+New Common Options in GMT 6.2:
+------------------------------
+
+#. :ref:`-w <gmt:The **-w** option>`: Convert selected coordinate to repeating cycles.
+
+New Modules in GMT 6.2:
+-----------------------
+
+#. :doc:`gmtbinstats`: Bin data and determine statistics per bin.
+
+New Core Module Features in GMT 6.2:
+------------------------------------
+
+#. :doc:`colorbar`: New **+x** and **+y** modifiers to the **-S** option for setting axis label and unit; Support
+   slanted annotations with **-S**.
+#. :doc:`events`: New **-Z** option to animate geodesy and seismology symbols; New **-H** option to enable text label
+   boxes; Support plotting lines as series of closely spaced circles.
+#. :doc:`gmtsplit`: New name for previous module splitxyz.
+#. :doc:`grd2xyz`: New modifier for the **-W** option to set the length unit used.
+#. :doc:`grdcut`: New **-F** option to clip a grid based on a polygon.
+#. :doc:`grdfft`: New **-Q** option for no wavenumber operations.
+#. :doc:`grdmath`: New **FISHER** and **VPDF** operators.
+#. :doc:`greenspline`, :doc:`grdinterpolate`: Enable writing of 3-D netCDF data cubes.
+#. :doc:`histogram`: New **-E** option for custom bar widths and optional shift; New **+b** modifier to **-C** to set
+   color based on the bin value.
+#. :doc:`legend`: New **-M** option to handle both hidden and given information.
+#. :doc:`makecpt`, :doc:`grd2cpt`: Simplify the addition of category labels to CPT files with **-F**.
+#. :doc:`movie`: New modifiers to **-L** and **-P** to enable drop-shadow and rounded rectangular boxes.
+#. :doc:`plot`, :doc:`plot3d`: New **-H** option to scale the symbol size as well as the symbol pen outline attributes;
+   Support sequential auto-colors for polygon fills or line pens.
+#. :doc:`subplot`: New **-D** option to accept previous default plot settings.
+#. :doc:`ternary`: Add support for drawing lines and polygons.
+#. :doc:`text`: New **-S** option to cast shade beneath a text box.
```

Comment:
We have also the hexagonal plots (sorry, don't remember exactly the details) which are an import improvement.

Reply:
Is this referring to the new gmtbinstats module? PR https://github.com/GenericMappingTools/gmt/pull/3950

Reply:
Ah, yes. What I had in mind was to emphasize ``..\test\gmtbinstats\hexbinning.ps``

Reply:
OK, I added that gmtbinstats supports both hexagonal and rectangular tiling.